### PR TITLE
Update after feedback from JAX

### PR DESCRIPTION
**Pull request by viboes**
Sorry, I have not finished all the feedback from JAX.

I'll try to fix the merge issue soon.

Any additional feedback is welcome.

### CODE REVIEW

**Comment by jfbastien** [comment]
Weird, GitHub thinks there's a merge conflict. I think you need to rebase and squash to one commit? It looks like there's history since November 2017 in this PR.

**Comment by viboes** [comment]
I've tried to merge but I have a big difference in the merged file :(
As if there where a end-of-line issue.

**Comment by viboes** [comment]
I don't know how to manage with the history.
I have just committed the file in my own repo and pushed a request

**Comment by jfbastien** [comment]
I'll untangle this over the weekend.

**Comment by jfbastien** [comment]
OK I took your PR and untangled the problems, and then committed here:

https://github.com/jfbastien/papers/commit/688ffa0af5196b5ccfd29a2f159900bcfa706c67

Main problem was DOS line endings confusing GitHub.

There also seems to be a problem with your repository. Usually you don't commit straight to master when you're working of a fork. Here's the type of workflow people usually follow:

https://gist.github.com/Chaser324/ce0505fbed06b947d962

Basically: fork a repo, work on a branch, then re-sync master from origin, rebase your branch onto master, push your branch to your GitHub fork, and then create the PR.

**Comment by jfbastien** [comment]
(closing since I committed the changes already)

**Comment by viboes** [comment]
Thanks JF.

I will try to follow this workflow for the next PR.

Vicente
